docs/src/pages/utils/popovers/AnchorPlayground.js

```diff
@@ -76,8 +72,8 @@ class AnchorPlayground extends React.Component {
     return (
       <div>
         <Button
-          ref={node => {
-            this.button = node;
+          buttonRef={elem => {
```

Comment:
We use `node` everywhere. What's wrong with the wording?

Reply:
That was an inadvertent change 🙂